openapi.json

```diff
@@ -0,0 +1,9995 @@
+            "x-nullable": true
+          },
+          "id": {
+            "description": "to be documented",
+            "readOnly": true,
+            "title": "id",
+            "type": "integer"
+          },
+          "master_accounts": {
+            "items": { "type": "integer" },
+            "type": "array"
+          },
+          "owner": { "items": { "type": "integer" }, "type": "array" },
+          "persons": { "items": { "type": "integer" }, "type": "array" },
+          "subject": {
+            "description": "Memo Subject Line / Heading",
+            "nullable": true,
+            "title": "subject",
+            "type": "string",
+            "x-nullable": true
+          },
+          "tags": { "items": { "type": "integer" }, "type": "array" },
+          "type": {
+            "description": "Memo Type",
+            "enum": [
+              "Billing",
+              "MDE AMLA",
+              "MDE other",
+              "Data Quality",
+              "Client Request",
+              "Compliance Request"
+            ],
+            "title": "type",
+            "type": "string"
+          }
+        },
+        "required": ["type"],
+        "type": "object"
+      },
+      "organization": {
+        "example": {
+          "changed_by": "changed_by",
+          "changed_on": "2000-01-23T04:56:07+00:00",
+          "created_by": "created_by",
+          "created_on": "2000-01-23T04:56:07+00:00",
+          "id": 0,
+          "name": "name",
+          "owner": "owner",
+          "users": [6, 6]
+        },
+        "properties": {
+          "changed_by": {
+            "description": "User name/email of the user who last changed the record via API",
+            "readOnly": true,
+            "title": "changed_by",
+            "type": "string"
+          },
+          "changed_on": {
+            "description": "Timestamp on which the record was last changed, automatically filled by the API",
+            "format": "date-time",
+            "readOnly": true,
+            "title": "changed_on",
+            "type": "string"
+          },
+          "created_by": {
+            "description": "User name/email of the user who inserted the record via API",
+            "readOnly": true,
+            "title": "created_by",
+            "type": "string"
+          },
+          "created_on": {
+            "description": "Timestamp on which the record was created, automatically filled by the API",
+            "format": "date-time",
+            "readOnly": true,
+            "title": "created_on",
+            "type": "string"
+          },
+          "id": {
+            "description": "ID of organisation which user belong to",
+            "readOnly": true,
+            "title": "id",
+            "type": "integer"
+          },
+          "name": {
+            "description": "Name of organisation which user belong to",
+            "title": "name",
+            "type": "string"
+          },
+          "owner": {
```

Comment:
Which type is this referring to? `person`, `master_account`, other?

Reply:
Same as above => User entity => all the audit fields (created by / changed by) will link to a User with an ID and a Username

Reply:
Got it. So we'll always consider owner part of the audit interface, pointing to a user id.